packages/web/src/fixtures/dev-kit/client.ts

```diff
@@ -163,3 +175,18 @@ export const holdersShare = async (amount: string, lockedups: string) => {
   }
   return undefined
 }
+
+export const createGetVotablePolicy = async () => {
+  const web3Client = newWeb3Client()
+  if (web3Client) {
+    // FIXME(@k3nt0w)
+    // return devKitClient.createGetVotablePolicy(getPolicySetContract(web3Client))()
+    return [
+      '0xCbC478710D3aCD2E7B94F58BB639f345af5f7869',
+      '0xCbC478710D3aCD2E7B94F58BB639f345af5f7869',
+      '0xCbC478710D3aCD2E7B94F58BB639f345af5f7869',
+      '0xCbC478710D3aCD2E7B94F58BB639f345af5f7869'
+    ]
```

Comment:
To check the operation of the Component, a fixed policy address is returned here.